init/pkg/initialize.go

```diff
@@ -0,0 +1,57 @@
+package pkg
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+const configVolume = "/etc/aerospike"
+
+func (initp *InitParams) ColdRestart() error {
+	if err := initp.makeWorkDir(); err != nil {
+		return err
+	}
+
+	// Copy required files to config volume for initialization.
+	if err := initp.copyTemplates("/configs", configVolume); err != nil {
+		return err
+	}
+
+	filesToCopy := [2]string{"/workdir/bin/akoinit", "/configs/features.conf"}
+	for _, file := range filesToCopy {
+		if _, err := os.Stat(file); err == nil {
```

Comment:
os.Stat command is used here to check if file exist or not before copying, otherwise cp command will fail. And in scripts also we were not failing the script if features.conf file was not there.

Reply:
Add a comment that we shouldn't fail if features.conf is not present. akoinit will always be there because it is getting checked in entrypoint.sh